Week 1 and Week 2 Challenge by Erika Nagai

Enagai-nagai opened the pull request:
I created week1 and week2 challenge files in the folder "post" instead of the folder "template". I hope you can merge them.

Enagai-nagai committed:
Week1 Submission

Enagai-nagai committed:
Merge pull request #1 from DACSS/template

Merging week 2 tutorials

Enagai-nagai committed:
Week1 challenge submittion

Enagai-nagai committed:
Challenge 1 and US input data

Merge branch 'template' of https://github.com/Enagai-nagai/Social_Networks_Spring_2023 into template

# Conflicts:
#	_templates/Week1_Challenge_Instructions.qmd

Enagai-nagai committed:
Week 1 and 2 challenge

Enagai-nagai committed:
Delete Week1_Challenge_Instructions.qmd

Enagai-nagai committed:
Delete Week1_Challenge_ErikaNagai.qmd

Enagai-nagai committed:
Delete Week2_Challenge_ErikaNagai.qmd

Enagai-nagai committed:
Week 1 and 2 submission by Erika Nagai

SidharthSaluja reviewed (commented):
FIle in the _templates folder should not be changed. Also I do want freeze files. Render your qmd files to generate freezer files. Please slack me or book appointment with me for any confusion

Enagai-nagai committed:
Freeze files

Enagai-nagai committed:
tutorial week1

I had accidentally deleted this file so I'm uploading it again

Enagai-nagai commented:
Thanks for the comment.
I added freeze files (rendered files) and also added a week1 tutorial file. I had mistakenly deleted it so I uploaded it again.
I hope it work.
